workflows/scripts/seed/bulk-download.py

```diff
@@ -0,0 +1,37 @@
+from database.models.workflow import Workflow
+from database.models.workflow_version import WorkflowVersion
+
+from platformics.util.seed_utils import SeedSession, DEFAULT_WORKFLOW_VERSIONS, TempCZIDWorkflowFile
+
+
+def main() -> str:
+    """
+    An idempotent seed script to create the minimum viable set of entities to run consensus genomes
+
+    It also creates entity and raw inputs to run the consensus genome workflow
+    """
+    session = SeedSession()
+
+    version = DEFAULT_WORKFLOW_VERSIONS["bulk-download"]
+    cg_workflow = session.create_or_fetch_entity(Workflow, name="bulk-download")
+    cg_workflow.default_version = version
+    cg_workflow.minimum_supported_version = version
+    session.add(cg_workflow)
+    session.commit()
+
+    workflow_uri = session.transfer_wdl("run.wdl", "bulk-download")
+    cg_workflow_version = session.create_or_fetch_entity(WorkflowVersion, version=version, workflow=cg_workflow)
+    cg_workflow_version.workflow_uri = workflow_uri
+    with TempCZIDWorkflowFile("manifest.yml", "bulk-download") as manifest_file:
```

Comment:
Is this in main branch already?

Reply:
yes I tested before merging then merged in workflows and updated here